### ISSUE TITLE
dbQuoteString() needs a method for Hive

### ISSUE THREAD

**Issue by yutannihilation** [r-dbi/odbc#184]
Hive uses C-style escape for string literals, so the result of this pseudo reprex should be `'it\'s ok'`. (Sorry for not providing the actual reprex...)

```r
# `con` is a Hive connection
DBI::dbQuoteString(con, "it's ok")
#> <SQL> 'it''s ok'
```

c.f.) https://cwiki.apache.org/confluence/display/Hive/Literals

To do this, is it necessary to define something like `odbcQuoteString()` generic function?

**Comment by yutannihilation** [comment]
Thanks! I will try.

**Comment by simonpcouch** [comment]
Closed in #424.☃️
